server.go

```diff
@@ -443,6 +443,7 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 			htlcswitch.DefaultLogInterval),
 		NotifyActiveChannel:   s.channelNotifier.NotifyActiveChannelEvent,
 		NotifyInactiveChannel: s.channelNotifier.NotifyInactiveChannelEvent,
+		RejectHTLC:            false,
```

Comment:
probably don't need this commit since it's the default

Reply:
fixed!

Reply:
@cfromknecht just realised think that was an old commit. 

Server needs to pull the bool flag from configuration 
```
RejectHTLC: cfg.RejectHTLC
```

This will then enable the `htlcswitch/switch.go - handlePacketForward` to reject onward htlcs according to the configuration flag when creating the `htlcswitch.Config{...}` in `server.go`.